utils/frame/try-runtime/cli/src/commands/follow_chain.rs

```diff
@@ -42,51 +46,177 @@ pub struct FollowChainCmd {
 	uri: String,
 }
 
+/// Start listening for with `SUB` at `url`.
+///
+/// Returns a pair `(client, subscription)` - `subscription` alone will be useless, because it
+/// relies on the related alive `client`.
+async fn start_subscribing<Header: DeserializeOwned>(url: &str) -> (Client, Subscription<Header>) {
+	let client = WsClientBuilder::default()
+		.connection_timeout(std::time::Duration::new(20, 0))
+		.max_notifs_per_subscription(1024)
+		.max_request_body_size(u32::MAX)
+		.build(url)
+		.await
+		.unwrap();
+
+	log::info!(target: LOG_TARGET, "subscribing to {:?} / {:?}", SUB, UN_SUB);
+
+	let sub = client.subscribe(SUB, None, UN_SUB).await.unwrap();
```

Comment:
would it be too hard to propagate the error?

Reply:
initially I left it as was before, but now the errors from both places in this function will be propagated :+1: